src/garage/tf/algos/npo.py

```diff
@@ -102,6 +102,8 @@ def __init__(self,
         self.policy = policy
         self._scope = scope
         self.max_episode_length = env_spec.max_episode_length
+        if self.max_episode_length == None:
+            raise ValueError("max_episode_length must not be None")
```

Comment:
we use single quotes in this repo. we have an automated tool called pre-commit which will check your PR for small issues like this and tell you how to fix them (or you can check the CI results below, which will also tell you what to fix).

See the guide here: https://garage.readthedocs.io/en/latest/user/preparing_a_pr.html and https://garage.readthedocs.io/en/latest/user/git_workflow.html#passing-the-pre-commit-hooks